mail_template_multi_company/security/mail_template.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+    <data>
```

Comment:
I think that the default rule in Odoo is to set ``ir.rule`` defined in security folder as "noupdate="1"" to allow user to change it by the UI.

https://github.com/odoo/odoo/blob/12.0/odoo/addons/base/security/base_security.xml#L7

Reply:
Hi @legalsylvain ,
Thank you for your review,

User can archive an ir.rule and create an alternative.

Reply:
@sbejaoui @legalsylvain is there a written OCA convention for noupdate in record rules?

Reply:
Hi @sbidoul. Well, not an OCA convention, AFAIK. for my side, just a recurrent Odoo rule, I observed. 
All the rule in Odoo are in noupdate="1" (and all the modules in OCA I use and / or developped).

ref : product module in V13 : (for exemple) 

https://github.com/odoo/odoo/blob/13.0/addons/product/security/product_security.xml#L32

My preference is to use this "convention". don't you think ?

I don't know who ping in Odoo team to know the reason of this choice.